### PR TITLE
North finding

### PR DESCRIPTION
**Pull request by AdityaMatam**
Added python node that subscribes to filtered odometry, and publishes the orientation of the rover.

### CODE REVIEW

File: Workspace/src/north_finder/north_finder.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    # Converts filtered odometry into a heading angle,
+    # and then publishes the angle
+    def odom_to_yaw(self, data):
+        quaternion = (
+            data.pose.pose.orientation.x,
+            data.pose.pose.orientation.y,
+            data.pose.pose.orientation.z,
+            data.pose.pose.orientation.w)
+        euler = tf.transformations.euler_from_quaternion(quaternion)
+        yaw = euler[2] / math.pi * 180
```
Comment (archielee) [inline Workspace/src/north_finder/north_finder.py]: do we know 0 yaw is north? and range is this number in

Comment (archielee) [reply Workspace/src/north_finder/north_finder.py]: @AdityaMatam ping

**Comment by AdityaMatam** [comment]
Added comments to make it more clear what it's returning. North is 0, and the angles range from -90 degrees to 270 degrees (imbalanced because I compensated for the fact that East was originally 0).

**Comment by archielee** [comment]
Looks good to me

**Pull request merged**